backend/src/openarchiefbeheer/destruction/api/viewsets.py

```diff
@@ -474,6 +488,55 @@ def get_permissions(self):
         return [permission() for permission in permission_classes]
 
 
+@extend_schema_view(
+    list=extend_schema(
+        tags=["Co-Reviews"],
+        summary=_("List co-reviews"),
+        description=_(
+            "List all the co-reviews that have been made for a destruction list."
+        ),
+    ),
+    create=extend_schema(
+        tags=["Co-Reviews"],
+        summary=_("Create co-review"),
+        description=_(
+            "Create a co-review for a destruction list. "
+            "Only a user currently assigned as co-reviewer to the destruction list can create a review."
+        ),
+    ),
+)
+class DestructionListCoReviewViewSet(
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    viewsets.GenericViewSet,
+):
+    serializer_class = DestructionListCoReviewSerializer
+    queryset = DestructionListCoReview.objects.all()
+    filter_backends = (DjangoFilterBackend,)
+    filterset_class = DestructionListCoReviewFilterset
+
+    def get_permissions(self):
+        if self.action == "create":
+            permission_classes = [IsAuthenticated & CanCoReviewPermission]
+        else:
+            permission_classes = [IsAuthenticated]
+        return [permission() for permission in permission_classes]
+
+    def get_queryset(self):
+        qs = super().get_queryset()
+        if self.action == "list":
+            # Return CoReview object which
+            # - associated DestructionList has no DestructionListReview
+            # - associated DestructionList has no ReviewResponse associated with its DestructionListReview
+            # - was created after the creation date of the ReviewResponse associated with the DestructionListReview
+            #   associated with the DestructionList
```

Comment:
What is the reason for looking at the response in addition to the review?

Reply:
My thoughts: the submission of the response starts a new review round, the review "just" complets it. In the current implementation it's visible to the record manager what co-reviewer assisted the review which I think is useful information.

Reply:
As discusses we leave this in for now.